README.md

```diff
@@ -205,6 +209,29 @@ working directory called `corrected`._
 * Resources for uds3 fvp forms are available [here](https://www.alz.washington.edu/NONMEMBER/UDS/DOCS/VER3/).
 
 
+
+Example Workflow
+----------------
+Once you have edited the `nacculator_cfg.ini` file with your api token and desired filters, you can get a filtered csv of the REDCap data with
+
+`python run_filters.py nacculator_cfg.ini`
+
+This will create a run folder with the current date that contains the csv and each iteration of filter, ending with `final_update.csv`.
+You will likely need to split apart the IVP and FVP visits.
+
+`bash split_ivp_fvp.sh <run_folder>/final_update.csv`
```

Comment:
Instead of `<run_folder>`, use a bash variable `$run_folder`. That way, if someone sets the variable (for example, `export run_foder=runs`), they could copy-and-paste your command.